librz/core/linux_heap_jemalloc.c

```diff
@@ -104,15 +104,15 @@ static bool GH(rz_resolve_jemalloc)(RzCore *core, char *symname, ut64 *symbol) {
 	free(path);
 	return false;
 #else
-	eprintf("[*] Resolving %s from libjemalloc.2... ", symname);
+	rz_cons_printf("[*] Resolving %s from libjemalloc.2... ", symname);
```

Comment:
@wargio @thestr4ng3r I think this is better as a eprintf ? It is not the "real" output of the command, it's just an info... maybe RZ_LOG_INFO or something like that. rz_cons_printf feels wrong though.

Reply:
RZ_LOG_INFO sounds good for me.

Reply:
should I modify all similar ones to RZ_LOG_INFO?

Reply:
Same, please remove the `[*]`

Reply:
done!
I found in the file, some printing functions with space after them and other without. Don't you mind this inconsistency?

Reply:
The ones commented out are all with spaces

Reply:
Just install `clang-format` and then run `./sys/clang_format.py` from the rizin root folder.

Reply:
I did run but no changes happened.

Reply:
just commit

Reply:
No changes in linux_heap_jemalloc.c file. It is up-to-date now